src/command_modules/azure-cli-sf/azure/cli/command_modules/sf/custom.py

```diff
@@ -0,0 +1,1480 @@
+    information.
+    :param str health_state: Possible values include: 'Invalid', 'Ok',
+    'Warning', 'Error', 'Unknown'
+    :param int ttl: The duration, in milliseconds, for which this health report
+    is valid. When clients report periodically, they should send reports with
+    higher frequency than time to live. If not specified, time to live defaults
+    to infinite value.
+    :param str description: The description of the health information. It
+    represents free text used to add human readable information about the
+    report. The maximum string length for the description is 4096 characters.
+    If the provided string is longer, it will be automatically truncated.
+    When truncated, the last characters of the description contain a marker
+    "[Truncated]", and total string size is 4096 characters. The presence of
+    the marker indicates to users that truncation occurred. Note that when
+    truncated, the description has less than 4096 characters from the original
+    string.
+    :param str sequence_number: The sequence number for this health report as a
+    numeric string. The report sequence number is used by the health store to
+    detect stale reports. If not specified, a sequence number is auto-generated
+    by the health client when a report is added.
+    :param bool remove_when_expired: Value that indicates whether the report is
+    removed from health store when it expires. If set to true, the report is
+    removed from the health store after it expires. If set to false, the report
+    is treated as an error when expired. The value of this property is false by
+    default. When clients report periodically, they should set this value to
+    false (default). This way, is the reporter has issues (eg. deadlock) and
+    can't report, the entity is evaluated at error when the health report
+    expires. This flags the entity as being in Error health state.
+    :param long timeout: The server timeout for performing the operation in
+    seconds. This specifies the time duration that the client is willing to
+    wait for the requested operation to complete. The default value
+    for this parameter is 60 seconds.
+    """
+
+    # TODO Move common HealthInformation params to _params
+
+    from azure.servicefabric.models.health_information import HealthInformation
+    from azure.cli.command_modules.sf._factory import cf_sf_client
+
+    info = HealthInformation(source_id, health_property, health_state, ttl,
+                             description, sequence_number, remove_when_expired)
+
+    sf_client = cf_sf_client(None)
+    sf_client.report_service_health(service_id, info, timeout)
+
+
+# pylint: disable=too-many-arguments
+def sf_report_partition_health(partition_id, source_id, health_property,
+                               health_state, ttl=None, description=None,
+                               sequence_number=None, remove_when_expired=None,
+                               timeout=60):
```

Comment:
Move the pylint disable argument in line. This way the disable statement won't leak into the file scope.
```
def sf_report_partition_health(partition_id,  # pylint: disable=too-many-arguments
                               source_id, health_property,
                               health_state, ttl=None, description=None,
                               sequence_number=None, remove_when_expired=None,
                               timeout=60):
```
For other places as well.